openshift-knative-operator/cmd/operator/kodata/ingress/0.23/0-networkpolicy-mesh.yaml

```diff
@@ -43,3 +43,18 @@ spec:
   ingress:
   - {}
 ---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-from-openshift-monitoring-ns
+  namespace: knative-serving
```

Comment:
This lacks the provider label, so it'll be applied even with kourier, which we don't want.

Reply:
Thank you! Updated.